src/actions/index.ts

```diff
@@ -15,7 +14,7 @@ export const RECEIVE_RECORDING_INFO = 'RECEIVE_RECORDING_INFO'
 
 export const ADD_RECORDING = 'ADD_RECORDING'
 export const DELETE_RECORDINGS = 'DELETE_RECORDINGS'
-export const SET_RECORDING_INFO = 'SET_RECORDING_INFO'
+export const DELETE_SORTINGS_FOR_RECORDINGS = 'DELETE_SORTINGS_FOR_RECORDINGS'
```

Comment:
When a recording is deleted, we also dispatch this action to delete all sortings associated with the deleted recordings.